### DISCUSSION TITLE
Multiple Snowflake Accounts?

### DISCUSSION THREAD

**Discussion by BenKentzer** [littleK0i/SnowKill#2] [Q&A] [answered]
Hello!

Is it possible to run multiple instances of SnowKill on a single machine? 
We have quite a few individual accounts, and I'm quite interested in looking at using this as part of our monitoring toolkit.

Also - I assume it's okay for use SQLite as the underlying database and that I just need to monkey about with the relevant scripts?

Thanks
Ben

**Comment by littleK0i** [comment, marked as answer]
Sure, you may run as many instances as you want. SnowKill is a basic Python script.

It should be relatively easy to use SQLite, if you prefer. Here is the Postgres storage class: https://github.com/littleK0i/SnowKill/blob/master/snowkill/storage/postgres_table.py

You may create your own storage class and implement only one function: `store_and_remove_duplicate`. It is going to be mostly copy-paste, with some minor adjustments.

No need to monkey-patch SnowKill, you may implement this class in your own code and straight up use it instead of Postgres storage.

**Comment by BenKentzer** [comment]
Just wanted to update - I have now got a working monitoring solution (so far on 3 of our accounts) using a combination of SnowKill and my own home-baked modules managing the emailing (we have to use a pre-configured account on the server) and the linkages into SQLite3.

Just need to get the scheduling sorted, and add the rest of our accounts... We have a few!
